### ISSUE TITLE
cat: ./dumpfile: No such file or directory

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sqlifuzzer/sqlifuzzer#1]
```
What steps will reproduce the problem?
 Running: 

./sqlifuzzer-0.5.sh -t http://DOMAIN -l /home/new/burplogreq -c 'MYCOOKIE' 
-snqr -W


What is the expected output? What do you see instead?

cat: ./dumpfile: No such file or directory

What version of the product are you using? On what operating system?

sqlifuzzer-0.5l.tgz 

Please provide any additional information below.

Can someone tell me why i am seeing many lines of this, is there something that 
i missed?

Look forward to hearing your comments

Cheers

```

Original issue reported on code.google.com by `del...@outlook.com` on 7 Aug 2012 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
good catch delist. 

this message is often seen when the server responses consist of only headers, 
i.e. no body. this could be a 302 relocate back to the login page as a result 
of a problem with a session cookie, or a 404 with no content.... but it could 
also be something completely different.

most of the time when ive seen 'cat: ./dumpfile: No such file or directory' 
messages i've been able to address the issue by troubleshooting the connection. 
i.e. are you sure the host is accessed over http and not https, etc, etc...

one option is to try using the -T switch and provide the URL for 
post-authentication page; if nothing else, you'll be able to see the response.

i will have a look at the response-handling code and see if this can be caught 
and handled better. something like: an [Empty Response] alert would be more 
useful than a bash error...
```

Original comment by `sqlifuz...@gmail.com` on 17 Sep 2012 at 9:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `sqlifuz...@gmail.com` on 20 Sep 2012 at 9:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 0.7 (currently svn only) Implemented better handling and reporting of 
responses with no content. 
```

Original comment by `sqlifuz...@gmail.com` on 29 Oct 2012 at 11:55
- Changed state: **Fixed**
